guardian/grid
# new react sort control

## What does this change?

The new sort control replaces the existing sort control found in the top bar. The current control displays as;

<img width="1504" alt="Screenshot 2024-04-03 at 10 02 39" src="https://github.com/guardian/grid/assets/128470622/c553702b-f068-479b-a16c-cc9247bf5f88">

The new sort control appears as;

<img width="1474" alt="Screenshot 2024-04-03 at 12 23 28" src="https://github.com/guardian/grid/assets/128470622/f8c6d43a-ef19-43b0-8169-0064ed96027f">

It currently offers the same range of sort options as are currently available but has the potential to be extended to offer a wider range of sort options.

## How should a reviewer test this change?

This should be tested by ensuring the selected sort option is reflected in the images url (old to new should show as 'oldest'). Testershould also ensure that the control responds correctly under refresh and when the logo is slected.

## Tested? Documented?
